src/styles/components/nav.scss

```diff
@@ -0,0 +1,49 @@
+.nav {
+  transition-duration: 0.3s;
+
+  &__list {
+    display: flex;
+    list-style: none;
+    margin: 0;
+    padding: 0;
+  }
+
+  &__item {
+    position: relative;
+    margin-right: 20px;
+
+    &:last-child {
+      margin-right: 0;
+    }
+  }
+
+  &__link {
+    display: block;
+    position: relative;
+    padding: 23px 0;
```

Comment:
The vertical padding for `.nav__link` is quite large. Ensure that this padding aligns with the design requirements. If the design specifies a smaller padding, you should adjust this value accordingly.